libraries/qr_make.py

```diff
@@ -0,0 +1,21 @@
+import qrcode
+
+
+def qr_make(web_link):
+    qr = qrcode.QRCode(
+        version=6,
+        error_correction=qrcode.constants.ERROR_CORRECT_H,
+        box_size=10,
+        border=4,
+    )
```

Comment:
Any auto-formatter tool for Python that you are aware of?

Reply:
Please don't address that issue in this PR

Reply:
Added check on PEP format style with Black package.